chart/compass/charts/connector/templates/deployment.yaml

```diff
@@ -51,9 +51,15 @@ spec:
               value: "{{ .Values.deployment.args.certificateValidityTime }}"
             - name: APP_CA_SECRET_NAME
               value: "{{ .Values.global.connector.secrets.ca.namespace }}/{{ .Values.global.connector.secrets.ca.name }}"
+            - name: APP_CA_SECRET_CERTIFICATE_KEY
+              value: "{{ .Values.global.connector.secrets.ca.certificateKey }}"
+            - name: APP_CA_SECRET_KEY_KEY
+              value: "{{ .Values.global.connector.secrets.ca.keyKey }}"
             {{ if .Values.deployment.args.attachRootCAToChain }}
             - name: APP_ROOT_CA_CERTIFICATE_SECRET_NAME
```

Comment:
Shouldn't this env name be changed to `APP_ROOT_CA_SECRET_NAME`? As this changed in config struct.

Reply:
Sure! Thanks for the catch